stdlib/private/StdlibCollectionUnittest/MinimalCollections.swift.gyb

```diff
@@ -1341,7 +1341,11 @@ public struct Defaulted${traversal}RangeReplaceableSlice<Element>
   }
 
   public func generate() -> MinimalGenerator<Element> {
-    return MinimalGenerator(Array(self))
```

Comment:
The original code uses an API requiring iteration in the implementation of the iteration API itself; this leads to infinite recursion with the PR's changes.